js/util/combineCriteria.ts

```diff
@@ -0,0 +1,31 @@
+import * as Collections from 'typescript-collections';
+import { PropExtractorCriteria, PropExtractorConfig } from './extractProps';
+
+export const getCombinedCriteria = (
+  componentName: string,
+  configs: PropExtractorConfig[]
+): PropExtractorCriteria => {
+  let remainingConfigs: PropExtractorConfig[] = [...configs];
+  let inclusionSet = new Collections.Set<string>();
+
+  while (remainingConfigs.length > 0) {
```

Comment:
Why not just `configs.forEach(config => {...})`?

Reply:
Yeah, I can't honestly say...

Reply:
Fixed